NSSNamespace/Initial_Bangazon_Site
# styling for cart view

## Changes
Cart View now has a left and right column.  The left column is for the the the name, price, and quantity of each item in the cart, as well as the total price.  The right column is for the payment dropdown, new payment button, and the submit order button.  The right column will be pulled below the left column at screen width 768px and below.

## To Test
- ```git fetch```
- ```git checkout ew-CartStyle```
- Select a Bangazon Customer and make sure there are items in your cart
- Navigate to the Cart View.  Make sure the styling looks good, and fits the description above
